web/app/js/components/util/CliQueryUtils.js

```diff
@@ -1,9 +1,11 @@
 import _compact from 'lodash/compact';
 import _isNil from 'lodash/isNil';
 
-const topRoutesDisplayOrder = [
+const topRoutesDisplayOrder = query => _compact([
   "namespace",
-];
+  "toResource",
+  _isNil(query.to_type) ? null : "toNamespace",
```

Comment:
i don't see a way to clear the "To Namespace" once it's been selected? not a big deal, maybe for a subsequent PR?

Reply:
yeah, I think it would be good to add a Clear button similar to Tap and Top; I'll follow up with another PR